features/media-queries.yml

```diff
@@ -0,0 +1,18 @@
+name: Media queries
+description: CSS Media queries allow you to provide different styles based on the user's device and settings. For example, `@media (width <= 700px) and (orientation:portrait)` selects screens less than 700 pixels wide and also are taller than they are wide.
+spec: https://drafts.csswg.org/mediaqueries-5/
+caniuse: css-mediaqueries
+group: media-queries
+status:
+  compute_from: css.at-rules.media
+compat_features:
+  - css.at-rules.media
+  - css.at-rules.media.color
+  - css.at-rules.media.grid
+  - css.at-rules.media.height
+  - css.at-rules.media.width
+  - css.at-rules.media.monochrome
+  - css.at-rules.media.orientation
+  - css.at-rules.media.aspect-ratio
```

Comment:
Should we capture these 2 media features, and maybe others, as separate features?
I think keeping height and width here is fine, this is what the vast majority of media query usage happens. But the other features feel sufficiently specific to be in their own web-features files.

Reply:
I opted not to create separate features for pre-2015 media queries, thinking it would be easy to split if users requested it, but I could go either way.